java/view-point-cloud-data-offline/src/main/java/com/esri/arcgisruntime/sample/viewpointclouddataoffline/MainActivity.java

```diff
@@ -59,24 +61,24 @@
     scene.setBaseSurface(surface);
 
     // add a PointCloudLayer to the scene by passing the URI of the scene layer package to the constructor
-    PointCloudLayer pointCloudLayer = new PointCloudLayer(
+    mPointCloudLayer = new PointCloudLayer(
         getExternalFilesDir(null) + getString(R.string.scene_layer_package_location));
 
     // add a listener to perform operations when the load status of the PointCloudLayer changes
-    pointCloudLayer.addDoneLoadingListener(() -> {
-      if (pointCloudLayer.getLoadStatus() == LoadStatus.LOADED) {
+    mPointCloudLayer.addDoneLoadingListener(() -> {
+      if (mPointCloudLayer.getLoadStatus() == LoadStatus.LOADED) {
         // add the PointCloudLayer to the operational layers of the scene
-        mSceneView.getScene().getOperationalLayers().add(pointCloudLayer);
+        mSceneView.getScene().getOperationalLayers().add(mPointCloudLayer);
```

Comment:
let's just add it to the scene outside of the done loading listener to keep it in scope